Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,223 @@
+.. _db_paplot:
+
+Pulse Average (PA) plot
+=======================
+
+When the stimulus set contains multiple pulses the pulse average plot allows to
+extract these single pulses to overlay, and average and visually inspect these.
```

Comment:
The purpose of the pulse averaging (PA) plot is to identify synaptic connections by pooling (via time alignment) and averaging the responses within and across sweeps to pulses (intended to produce action potentials) on a single headstage. It repeats this process for all headstages with pulses (action potentials).